src/components/Links/Links.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@edx/paragon';
+
+import './Links.scss';
+
+function Links(props) {
+  const { title, links } = props;
+
+  function handleLinkClick(e) {
+    e.preventDefault();
+    // TODO: exact functionality TBD
+  }
+
+  const linkItems = links.map(link => (
+    <li key={link.href}>
+      <Icon className={['fa', 'fa-file', 'text-link-icon']} />
+      <a href={link.href} onClick={handleLinkClick}>{link.title}</a>
```

Comment:
These links will be external links provided by an institution. We should add support for opening (or always open) these links in a new window, with `target="_blank" rel="noopener noreferrer"`.